src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -221,9 +243,10 @@ public void perform(
     LOGGER.log(
         Level.INFO,
         String.format(
-            "GKE Deploying, projectId: %s cluster: %s zone: %s", projectId, clusterName, zone));
+            "GKE Deploying, projectId: %s cluster: %s location: %s",
+            projectId, clusterName, getLocation()));
```

Comment:
@craigatgoogle If a user has an old configuration and tries to run their build, this call to `getLocation()` needs to call `setupLocation()` in order to get switched over from zone to location. We should do one release with this intermediate, then one at a later date with zone entirely removed, maybe when we're ready for release 1.0